test/e2e/cluster_extension_install_test.go

```diff
@@ -377,13 +373,10 @@ func TestClusterExtensionBlockInstallNonSuccessorVersion(t *testing.T) {
 	t.Log("By eventually reporting a successful installation")
 	require.EventuallyWithT(t, func(ct *assert.CollectT) {
 		assert.NoError(ct, c.Get(context.Background(), types.NamespacedName{Name: clusterExtension.Name}, clusterExtension))
-		assert.Equal(ct,
-			&ocv1alpha1.ClusterExtensionResolutionStatus{Bundle: ocv1alpha1.BundleMetadata{
-				Name:    "prometheus-operator.1.0.0",
-				Version: "1.0.0",
-			}},
-			clusterExtension.Status.Resolution,
-		)
+		cond := apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1alpha1.TypeProgressing)
+		require.NotNil(ct, cond)
+		assert.Equal(ct, ocv1alpha1.ReasonSuccess, cond.Reason)
+		assert.Contains(ct, cond.Message, "resolved to")
```

Comment:
Same thing regarding progressing condition check already existing :)

Reply:
fixed in https://github.com/operator-framework/operator-controller/commit/febe76e179455d20c593b3f5956a770835afcedc